scripts/pic32pinctrl.py

```diff
@@ -0,0 +1,283 @@
+# Copyright (c) 2024 Microchip
```

Comment:
You should keep original copyright with the new one, even when you rename a file.